pkg/impl/libraptorq/swig/stdint.swg

```diff
@@ -1,12 +1,26 @@
 %{
 #include <stdint.h>
+#include <cstdint>
```

Comment:
Is this file auto-generated by swig?

Reply:
Nope.  `stdint.swg` is our own utility SWIG module that establishes mapping between 
C++ `uintXX_t`/`intXX_t` types and Go `uintXX`/`intXX` types.